app/models/user.rb

```diff
@@ -12,9 +12,7 @@ class User < ActiveRecord::Base
     :timeoutable,
     :trackable,
     :two_factor_authenticatable,
-    :omniauthable,
     authentication_keys: [:email],
```

Comment:
linters are getting mad at this trailing comma

(separate discussion: our JS linters *require* trailing commas but our Ruby linters *disallow* it? very confusing IMO, and I really like keeping trailing commas to minimize diff noise....)